avocado_vt/vt_agent/src/avocado_vt/agent/managers/resource_backings/network/tap/tap_port_backing.py

```
@@ -0,0 +1,89 @@
+        :rtype: dict.
+        """
+        if not self.tap_ifname:
+            self.tap_ifname = "tap_" + utils_misc.generate_random_string(8)
+        try:
+            self.tap_fd = tap.open_tap("/dev/net/tun", self.tap_ifname, vnet_hdr=True)
+            self.tap_fd = self.tap_fd.split(":")
+            interface.bring_up_ifname(self.tap_ifname)
+            bridge.add_to_bridge(self.tap_ifname, self.switch)
+        except Exception as e:
+            self.release_resource(network_connection)
+            raise e
```

Ruff (0.14.10)

Use `raise` without specifying exception name

Remove exception name

(TRY201)